DataManagementSystem/Utilities/DMSHelpers.py

```diff
@@ -186,25 +191,32 @@ def getSiteSEMapping( self ):
     return S_OK( siteSEMapping )
 
   def getSites( self ):
+    """ Get the list of known sites """
     self.getSiteSEMapping()
     return sorted( self.siteSet )
 
-  def getTiers( self, withStorage = False, tier = None ):
-    sites = sorted( self.getShortSiteNames( withStorage = withStorage, tier = tier ).values() )
+  def getTiers( self, withStorage=False, tier=None ):
+    """ Get the list of sites for a given (list of) Tier level """
+    sites = sorted( self.getShortSiteNames( withStorage=withStorage, tier=tier ).values() )
     if sites and isinstance( sites[0], list ):
       # List of lists, flatten it
       sites = [s for sl in sites for s in sl]
     return sites
 
-  def getShortSiteNames( self, withStorage = True, tier = None ):
+  def getShortSiteNames( self, withStorage=True, tier=None ):
+    """ Create a directory of short site names pointing to full site names """
     siteDict = {}
     result = self.getSiteSEMapping()
     if result['OK']:
       for site in self.siteSEMapping[LOCAL] if withStorage else self.siteSet:
         grid, shortSite, _country = site.split( '.' )
-        if isinstance( tier, ( int, long ) ) and ( grid != 'LCG' or gConfig.getValue( '/Resources/Sites/%s/%s/MoUTierLevel' % ( grid, site ), 999 ) != tier ):
+        if isinstance( tier, ( int, long ) ) and \
+        ( grid != 'LCG' or
+          gConfig.getValue( '/Resources/Sites/%s/%s/MoUTierLevel' % ( grid, site ), 999 ) != tier ):
           continue
-        if isinstance( tier, ( list, tuple, dict, set ) ) and ( grid != 'LCG' or gConfig.getValue( '/Resources/Sites/%s/%s/MoUTierLevel' % ( grid, site ), 999 ) not in tier ):
+        if isinstance( tier, ( list, tuple, dict, set ) ) and \
+        ( grid != 'LCG' or
+          gConfig.getValue( '/Resources/Sites/%s/%s/MoUTierLevel' % ( grid, site ), 999 ) not in tier ):
```

Comment:
Did we live with this bug until now?

Reply:
What bug ? It is strictly the same no ?

Reply:
I agree ;-)

Reply:
Mmm... what did I see? I don't remember... forget it, sorry for the noise!